src/main/java/mate/academy/bookstore/dto/user/UserResponseDto.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.bookstore.dto.user;
+
+import lombok.Data;
+
+@Data
+public class UserResponseDto {
+    private Long id;
```

Comment:
see the example of response 

{
  "id": 1,
  "email": "john.doe@example.com",
  "firstName": "John",
  "lastName": "Doe",
  "shippingAddress": "123 Main St, City, Country"
}